.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,26 @@
+Make sure you have checked _all_ steps below.
+
+### Jira
+
+- [ ] My PR addresses the following [Parquet Jira](https://issues.apache.org/jira/browse/PARQUET/) issues and references them in the PR title. For example, "PARQUET-1234 My Parquet PR"
+  - https://issues.apache.org/jira/browse/PARQUET-XXX
+  - In case you are adding a dependency, check if the license complies with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
+
+### Tests
+
+- [ ] My PR adds the following unit tests __OR__ does not need testing for this extremely good reason:
+
+### Commits
+
+- [ ] My commits all reference Jira issues in their subject lines, and I have squashed multiple commits if they address the same issue. In addition, my commits follow the guidelines from "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)":
+  1. Subject is separated from body by a blank line
+  1. Subject is limited to 50 characters (not including Jira issue reference)
```

Comment:
(nit) We don't follow this specific practice. You can leave the recommendation though, but since we always copy JIRA titles verbatim, it's more like a recommendation for the JIRA title.

Reply:
I'll leave it in for now. I think it is nice to have a clear and concise title.